test/qta/logical_operators.qta

```diff
@@ -0,0 +1,28 @@
+start default
+
+  define application "test"
+
+    uses substance "test"
+      initial charge with 1 kg / unit for manufacture
+      define testA as 1
+      define testB as 0
+      define testC as 2
+      # Test AND: 1 and 0 = 0 (false), so should use else branch (30)
+      set manufacture to 100 if testA and testB else 30 endif mt during year 1
+      # Test OR: 1 or 0 = 1 (true), so should use if branch (50)
+      set manufacture to 50 if testA or testB else 20 endif mt during year 2
+      # Test XOR: 1 xor 2 = 0 (both are truthy), so should use else branch (40)
```

Comment:
Thank you! Can you please add an additional test which checks that this does operator precedence correctly with parentheses?